sections/Stories.tsx

```diff
@@ -0,0 +1,196 @@
+`;
+
+const Stories = ({ stories, heading }: StoriesProps) => {
+  const [currentSlide, setCurrentSlide] = React.useState(0);
+  const [sliderRef, slider] = useKeenSlider<HTMLDivElement>({
+    initial: 0,
+    loop: true,
+    slidesPerView: 2,
+    mode: "free-snap",
+    spacing: 36,
+    centered: true,
+    breakpoints: {
+      // TODO - change hardcoded value to theme.breakpoints.lg
+      "(max-width: 992px)": {
+        slidesPerView: 1,
+      },
+    },
+    slideChanged(s) {
+      setCurrentSlide(s.details().relativeSlide);
+    },
+  });
+  return (
+    <>
+      <Container>
+        <Row className={"mb-4 text-center"}>
+          <Col xs={12}>
+            <Heading>{heading}</Heading>
+          </Col>
+        </Row>
+      </Container>
+      <StoryContainer>
+        <div ref={sliderRef} className="keen-slider">
+          {stories.map((story: StoryProps) => (
+            <StoryCard key={story.name} className="keen-slider__slide p-5 mb-2">
+              <Row className={"my-auto pb-sm-5"}>
+                <Col sm={12} md={4} className={"text-center my-auto"}>
+                  <HexHeadshot
+                    alt={"Hacker Stories - " + story.name}
+                    src={"img/headshots/" + story.headshot}
+                  />
+                  <Name className="mt-4">{story.name}</Name>
+                  <School>{story.school}</School>
+                </Col>
+                <Col sm={12} md={8} className={"my-auto"}>
+                  <div className="ms-md-4 ms-lg-4 ms-xl-4">
+                    <Quote className="pb-3 pt-4 pt-md-0 pt-lg-0 pt-xl-0 ">
+                      {story.quote}
+                    </Quote>
+                    <Link href={story.link}>
+                      {story.name.split(" ")[0]}&apos;s project: {story.project}
+                    </Link>
+                  </div>
+                </Col>
+              </Row>
+            </StoryCard>
+          ))}
+        </div>
+        {slider && (
+          <Row>
+            <div className="d-flex justify-content-center">
+              <Arrow
+                onClick={(e) => {
+                  e.stopPropagation();
+                  slider.prev();
+                }}
+                src="assets/left.png"
+              />
+              <Arrow src="assets/center.png" />
+              <Arrow
+                onClick={(e) => {
+                  e.stopPropagation();
+                  slider.next();
+                }}
+                src="assets/right.png"
+              />
+            </div>
+            <div className="d-flex justify-content-center">
+              {Array.from(Array(slider.details().size).keys()).map((idx) => {
+                return (
+                  <>
+                    <Dot
+                      key={idx}
+                      onClick={() => {
+                        slider.moveToSlideRelative(idx);
+                      }}
+                      className={currentSlide === idx ? "" : "opacity-2"}
```

Comment:
since the `opacity-2` class is only being used here (as far as i can tell), i think it'd be cleaner to pass an `active` prompt to `Dot` and handle the opacity accordingly within the styling (plus that way we can keep the `Stories`-related styling in the same file

Reply:
yeee I agree that's probably the best way to do it